src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -102,6 +102,46 @@ private void SetDownloadQPResult(bool result)
                 It.IsAny<CancellationToken>())).Returns(Task.FromResult(result));
         }
 
+        [TestMethod]
+        public async Task BindingWorkflow_SaveServerExclusions_Success()
+        {
+            // Arrange
+            ConfigurableProgressController controller = new ConfigurableProgressController();
+            var notifications = new ConfigurableProgressStepExecutionEvents();
+
+            SetSaveServerExclusionsResult(true);
+
+            // Act
+            await testSubject.SaveServerExclusionsAsync(controller, notifications, CancellationToken.None)
+                .ConfigureAwait(false);
+
+            // Assert
+            controller.NumberOfAbortRequests.Should().Be(0);
```

Comment:
Perhaps it's also worth adding Verify that `SaveServerExclusionsAsync` was indeed called? same in the other test